src/main/java/twilightforest/block/ForceFieldBlock.java

```diff
@@ -59,75 +65,91 @@ public class ForceFieldBlock extends Block implements SimpleWaterloggedBlock, En
 
 	public ForceFieldBlock(BlockBehaviour.Properties properties) {
 		super(properties);
-		this.registerDefaultState(this.getStateDefinition().any().setValue(WATERLOGGED, false)
+		this.registerDefaultState(this.getStateDefinition().any().setValue(WATERLOGGED, false).setValue(AXIS, Direction.Axis.Y)
```

Comment:
??
https://github.com/TeamTwilight/twilightforest/blob/1.20.1/src/main/java/twilightforest/block/ForceFieldBlock.java#L61

Reply:
There's no AXIS property in the force codebase.

Reply:
In fact, this entire class doesn't match at all.

Reply:
There must be full parity with the 1.20.1 codebase.